src/Microsoft.VisualStudio.AppDesigner/Microsoft.VisualStudio.AppDesigner.vbproj

```diff
@@ -1,17 +1,21 @@
 ﻿<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
 <Project Sdk="Microsoft.NET.Sdk">
+
   <Import Project="..\..\build\import\VisualStudioDesigner.props" />
+
   <PropertyGroup>
     <!-- TODO: Function doesn't return a value on all code paths (https://github.com/dotnet/project-system/issues/2592) -->
     <NoWarn>$(NoWarn);42353;NU5125</NoWarn>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
     <TargetFramework>net472</TargetFramework>
+    <IsProjectSystem>false</IsProjectSystem>
```

Comment:
This `IsProjectSystem` property caught my attention. Is there a clearer name?

Reply:
The intent is to differentiate between the project system projects (`Managed.dll` and `Managed.VS.dll`) versus the designer projects (`Editors.dll` and `AppDesigner.dll`). Any ideas for a better name? Right now, the OptProf test used only works for the project system projects. So, we can't have that running for the others.

Reply:
I have changed this to `IsManagedProjectSystemProject`. That shouldn't clash with anything that might exist and is more specific.

Reply:
Thanks, that's clearer somehow.